app/config.c

```diff
@@ -214,6 +215,8 @@ static void parse_option(const char *option, const char *params)
         } else if (strncmp(params, "rr", 3) == 0 || strncmp(params, "one", 4) == 0) {
             cpu_mode = ONE;
         }
+    } else if (strncmp(option, "dark", 5) == 0) {
+        dark_mode = true;
```

Comment:
If we want, we could make it more future proof with a very little effort - look for option "color" or "color_template "here and pass the `param` to set_screen_palette(), so it can switch to any matching entry if it can find one...

Reply:
That's unnecessary efforts right now. Users just requested a black background.